website/content/terminal/usage/intros/stocks/options/index.md

```diff
@@ -1,268 +1,158 @@
+|unu |Unusual options activity for S&P 500 stocks. |
+|voi |Plot the volume and open interest together. |
+|vol |Plot the volume. |
+|vsurf |3-D volatility surface chart. |
 
-### How to use
+### Examples
 
-Navigate to the menu by typing `options`, from the `Stocks` menu, and then pressing enter. Alternatively, absolute path navigation can jump straight there, from anywhere. `/stocks/options`
+The examples here will start by entering the Options menu without a ticker symbol loaded.
 
-<img width="800" alt="image" src="https://user-images.githubusercontent.com/46355364/218986539-fa5a214d-b9d3-48fe-9e60-2082aa79bf79.png"></img>
+```console
+/stocks/options
+```
 
-By default, the Terminal loads a ticker using <a href="https://developer.tradier.com/" target="_blank" rel="noreferrer noopener">Tradier</a> as the source. Sign up for a free developer account and then enter that token using the <a href="/terminal/usage/guides/api-keys" target="_blank" rel="noreferrer noopener">`Keys menu`</a>. Alternatively, there is a choice to use yFinance data sets by attaching the argument as shown below. Help dialogues are displayed for any command by adding `-h` to the string. It is worth noting that this load command is different than the load command elsewhere.
+#### unu
 
-````
-(🦋) /stocks/options/ $ load -h
-usage: load [-t TICKER] [-h] [--source {YahooFinance,Tradier,Nasdaq,Intrinio}]
+Unusual options are described as those having a very high volume/open interest ratio.  This function is built using the data provided by [fdscanner.com](https://fdscanner.com).  By default, the top 20 options with the highest vol/OI ratio are returned as a table.
 
-Load a ticker into option menu
+```console
+unu
+```
 
-optional arguments:
-  -t TICKER, --ticker TICKER
-                        Stock ticker (default: None)
-  -h, --help            show this help message (default: False)
-  --source {YahooFinance,Tradier,Nasdaq,Intrinio}
-                        Data source to select from (default: YahooFinance)
+Ticker   | Exp        |   Strike | Type   |   Vol/OI |   Vol |   OI |   Bid |   Ask |
+|:---------|:-----------|---------:|:-------|---------:|------:|-----:|------:|------:|
+| TFC      | 2023-05-19 |     34   | Call   |    127.2 | 15904 |  125 |  0.4  |  0.5  |
+| SCHW     | 2023-06-02 |     40   | Put    |     55.5 | 11996 |  216 |  0.28 |  0.34 |
+| TSLA     | 2023-05-05 |    149   | Put    |     49.3 | 57454 | 1165 |  2.93 |  2.95 |
+| TSLA     | 2023-04-28 |    157.5 | Call   |     46.8 | 99677 | 2132 |  1.39 |  1.4  |
+| USB      | 2023-05-19 |     32   | Call   |     44.9 |  6692 |  149 |  1.35 |  1.4  |
+| NOC      | 2023-05-19 |    440   | Put    |     43.8 |  4420 |  101 |  6    |  7.2  |
+| GM       | 2023-06-02 |     28   | Put    |     43.1 |  5867 |  136 |  0.32 |  0.35 |
+| AAPL     | 2023-05-19 |    167.5 | Call   |     41.6 | 10888 |  262 |  2.91 |  2.93 |
+| ATVI     | 2023-07-21 |     85   | Call   |     36.5 |  5772 |  158 |  1.59 |  1.71 |
```

Comment:
Not sure I see the need for this table.  Takes a while to scroll through then you have another screenshot anyways.

Reply:
Removed the table.